src/clj/datasplash/core.clj

```diff
@@ -620,7 +621,9 @@ Example:
   ([coll options ^Pipeline p]
    (let [opts (merge {:coder (make-nippy-coder)}
                      (assoc options :label :generate-input))
-         ptrans (Create/of coll)]
+         ptrans (if (empty? coll)
+                  (Create/empty (make-nippy-coder))
```

Comment:
Should we reuse `(:coder opts)` instead of `(make-nippy-coder)` here?

Reply:
Yes you are right, solved